### PR TITLE
REFO-24,32 Employer and Admin Job Viewing Pages

### PR DESCRIPTION
**Pull request by kendebacker**

### CODE REVIEW

File: src/pages/AddJob.jsx
Comment (nehakumari2307) [inline src/pages/AddJob.jsx:34]: Can you please remove the admin piece as well from AddJob. As discussed yesterday we don't need jobs to be tied up to a particular person.

Comment (kendebacker) [reply src/pages/AddJob.jsx]: @nehakumari2307 done!

File: src/pages/AdminJobPostings.jsx
Comment (makersmelx) [inline src/pages/AdminJobPostings.jsx:31]: first, it's good for now. 

for style consistence, it could be better to use `<p className="font-bold"></p>`. the point is to apply  [font weight](https://tailwindcss.com/docs/font-weight) or [font-size](https://tailwindcss.com/docs/font-size) and so like. 

Same for  `<h2>` things

**Review by makersmelx — APPROVED** [review]

**Comment by makersmelx** [comment]
For Tianchi's good, please merge this after he merges his two. @kendebacker @KiLiG-fcx

File: src/main.jsx
Comment (makersmelx) [inline src/main.jsx:41]: FYI, these paths will be modified by someone else when the relevant page has been done

File: src/pages/AdminJobPostings.jsx
Comment (kendebacker) [reply src/pages/AdminJobPostings.jsx:31]: Ah I see, yep will do

**Pull request merged**
